core/src/main/java/hudson/util/TimeUnit2.java

```diff
@@ -63,7 +67,12 @@
  *
  * @since 1.5
  * @author Doug Lea
+ * @deprecated use {@link TimeUnit}. (Java 5 did not have all the units required, so TimeUnit2 was introduced because it
+ * had better conversion until Java 6 went out.)
  */
+@Deprecated
+@RestrictedSince("TODO")
+@Restricted(NoExternalUse.class)
```

Comment:
A restriction is inappropriate here. `@Deprecated` is appropriate and suffices. The reason for the existence of `@Restricted(NoExternalUse.class)` is to allow access from another package within the same source root, similar to `public` in a nonexported package in Java 9.